src/lib/features/export-import-toggles/import-file-reader.ts

```diff
@@ -0,0 +1,8 @@
+import * as fs from 'fs';
+
+export const readFile: (file: string) => Promise<string> = async (file) =>
```

Comment:
we can call it read-file.ts since it's a generic utility. Also we may use more high level promise based API (https://nodejs.org/en/learn/manipulating-files/reading-files-with-nodejs). Just need to check if out node version supports it

Reply:
even more clear (and in preparation of esm) `import * as fs from 'node:fs'` I know biome sometimes warns about specfiying import.

Reply:
Yeah I'll add a separate PR with this and a e2e test